kubectl-fdb/cmd/remove.go

```diff
@@ -1,5 +1,5 @@
 /*
- * remove.go
```

Comment:
Since we're not renaming the file, I don't think we want to change this in the header.

Reply:
Thanks, that was the refactoring. Let me fix that.